1-js/02-first-steps/01-hello-world/1-hello-alert/task.md

```diff
@@ -1,12 +1,11 @@
-importance: 5
+Trudność: 5
```

Comment:
In main repo I found this `Please don't translate "importance" (and other top metadata).`